source/deployment.rst

```diff
@@ -0,0 +1,66 @@
+.. highlight:: bash
+
+Deployment
+==========
+
+Building a Tagged Version
+--------------------------
+
+Tagging
+=======
+
+Generally, we increment the micro version and create a new tag.::
+
+   git tag x.y.z
```

Comment:
Now that we've started this, maybe you could find the time to push a commit
on top of it?

On Wed, Mar 11, 2015 at 11:00 AM Thomas A Caswell <notifications@github.com>
wrote:

> In source/deployment.rst
> <https://github.com/NSLS-II/docs/pull/16#discussion_r26219183>:
>
> > @@ -0,0 +1,66 @@
> > +.. highlight:: bash
> > +
> > +Deployment
> > +==========
> > +
> > +Building a Tagged Version
> > +--------------------------
> > +
> > +Tagging
> > +=======
> > +
> > +Generally, we increment the micro version and create a new tag.::
> > +
> > +   git tag x.y.z
>
> Tags should be annotated so that git describe behave correctly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NSLS-II/docs/pull/16/files#r26219183>.
>